cleanlab/count.py

```diff
@@ -510,6 +553,104 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
+
+    calibrate : bool, default = True
+        Calibrates confident joint estimate P(label=i, true_label=j) such that
+        np.sum(cj) == len(labels) and np.sum(cj, axis = 1) == np.bincount(labels).
+
+    return_indices_of_off_diagonals: bool, default = False
+        If true returns indices of examples that were counted in off-diagonals
+        of confident joint as a baseline proxy for the label issues. This
+        sometimes works as well as filter.find_label_issues(confident_joint).
+
+    Returns
+    -------
+    confident_joint_counts : np.ndarray
+      An array of shape ``(num_classes,2, 2)`` representing the confident joint, the matrix used for identifying label issues, which
+      estimates a confident subset of the joint distribution of the noisy and true labels, ``P_{noisy label, true label}``.
+      Entry ``(c, j, k)`` in the matrix is the number of examples in a one-vs-rest class confidently counted into the pair of ``(class c, noisy label=j, true label=k)`` classes.
+
+    Note: if `return_indices_of_off_diagonals` is set as True, this function instead returns a tuple `(confident_joint_counts, indices_off_diagonal)`
+    where `indices_off_diagonal` is a list of arrays (one per class) and each array contains the indices of examples counted in off-diagonals of confident joint for that class.
+    """
+
+    num_classes = get_num_classes(labels=labels, pred_probs=pred_probs)
+    try:
+        y_one = int2onehot(labels)
+    except TypeError:
+        raise ValueError(
+            "wrong format for labels, should be a list of list[indices], please check the documentation in find_label_issues for further information"
+        )
+    confident_joint_list: np.ndarray = np.ndarray(shape=(num_classes, 2, 2), dtype=np.int64)
+    indices_off_diagonal = []
+    for class_num in range(0, num_classes):
+        pred_probabilitites = _binarize_pred_probs_slice(pred_probs, class_num)
+        if return_indices_of_off_diagonals:
+            cj, ind = compute_confident_joint(
+                labels=y_one[:, class_num],
+                pred_probs=pred_probabilitites,
+                multi_label=False,
+                thresholds=thresholds,
+                calibrate=calibrate,
+                return_indices_of_off_diagonals=return_indices_of_off_diagonals,
+            )
+            indices_off_diagonal.append(ind)
+        else:
+            cj = compute_confident_joint(
+                labels=y_one[:, class_num],
+                pred_probs=pred_probabilitites,
+                multi_label=False,
+                thresholds=thresholds,
+                calibrate=calibrate,
+                return_indices_of_off_diagonals=return_indices_of_off_diagonals,
+            )
+        confident_joint_list[class_num] = cj
+
+    if return_indices_of_off_diagonals:
+        return confident_joint_list, indices_off_diagonal
+
+    return confident_joint_list
+
+
 def estimate_latent(
```

Comment:
Should this support multi-label as well?
find label issues doesn't call this with multilabal

Reply:
same Doubt for estimate_confident_joint_and_cv_pred_proba

Reply:
If these are never ever called by any configuration of `find_label_issues()`, then let's not support multilabel for these.  That said, can you add to the docstrings (for every such function) that:  

Multi-label classification is not supported in this method.

Reply:
Added "Multi-label classification is not supported in this method." to a couple of function docstrings